spec/factories/songs.rb

```diff
@@ -3,5 +3,6 @@
     name 'Dirty Diana'
     artist
     spotify_uri { SecureRandom.hex(10) }
+    plays 3001
```

Comment:
So, this is a great sport for a feature of FactoryGirl called `traits`.

You can create a "popular" song and an "unpopular" song as traits.

The use the factory and traits in your testcases.

See: https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#traits